webui/react/src/components/WorkspaceCreateModal.tsx

```diff
@@ -455,26 +504,40 @@ const WorkspaceCreateModalComponent: React.FC<Props> = ({ onClose, workspaceId }
         });
       }
     }
-  }, [form, workspaceId, canModifyAUG, canModifyCPS, resourceManagers]);
+  }, [form, canModifyAUG, canModifyCPS, workspaceId, loadedResourceQuotaList, resourceManagers]);
 
-  return (
-    <Modal
-      cancel
-      size="medium"
-      submit={{
-        form: idPrefix + FORM_ID,
-        handleError,
-        handler: handleSubmit,
-        text: 'Save Workspace',
-      }}
-      title={`${workspaceId ? 'Edit' : 'New'} Workspace`}
-      onClose={() => {
-        initFields(undefined);
-        onClose?.();
-      }}>
-      {modalContent}
-    </Modal>
-  );
+  if ((workspace && !canModifyWorkspace({ workspace })) ?? false) {
+    return (
+      <Modal
+        size="medium"
+        title="Workspace Config"
```

Comment:
if changing the title is the only reason to have a conditional statement here, we could just do that with the value passed to the `title` prop -- actually, this is already being done based on `workspaceId` below, so that could be changed to display "Workspace Config" instead of "Edit Workspace", if that's the intent.

Reply:
I also wanted to remove the submit option, with Edit it would show "Cancel" and "Save" with Workspace Config I just want an "OK" or something similar because they are not saving anything